.github/workflows/os_x_staticbuild.yml

```diff
@@ -8,18 +8,40 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
+
+      - name: Compilation cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.ccache
+          # We include the commit sha in the cache key, as new cache entries are
+          # only created if there is no existing entry for the key yet.
+          key: ${{ runner.os }}-ccache-${{ github.sha }}
```

Comment:
any way to share the cache across commits?

Reply:
Yes, via the restore-keys below. It's based on common prefix